tests/fast-integration/skr-test/README.md

```diff
@@ -0,0 +1,92 @@
+# SKR test
+
+This test covers SKR (SAP Kyma Runtime).
+
+## File Structure consists of the following parts:
+- **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
+- **oidc** folder contains the OIDC related tests
+- **commerce-mock** folder consists of two scripts:
+  1. **index.js** contains the commerce mock tests suite
+  2. **prep.js** can be called in stand-alone mode for commerce mock resources preparation
+
+##Run the SKR test
+
+Before you run the test make sure you prepare the `.env` file based on the `.env.template`:
+```
+INSTANCE_ID
+SKIP_PROVISIONING
+KEB_HOST=
+KEB_CLIENT_ID=
+KEB_CLIENT_SECRET=
+KEB_GLOBALACCOUNT_ID=
+KEB_SUBACCOUNT_ID=
+KEB_USER_ID=
+KEB_PLAN_ID=
+GARDENER_KUBECONFIG=
+
+COMPASS_HOST=
+COMPASS_CLIENT_ID=
+COMPASS_CLIENT_SECRET=
+COMPASS_TENANT=
+
+KCP_KEB_API_URL=
+KCP_GARDENER_NAMESPACE=
+KCP_OIDC_ISSUER_URL=
+KCP_OIDC_CLIENT_ID=
+KCP_OIDC_CLIENT_SECRET=
+KCP_TECH_USER_LOGIN=
+KCP_TECH_USER_PASSWORD=
+KCP_MOTHERSHIP_API_URL=
+KCP_KUBECONFIG_API_URL=
+
+AL_SERVICE_KEY= #must be a cloud foundry service key with the info about `UAA (User Account and Authentication)`. Learn more about [Managing Service Keys in Cloud Foundry](https://docs.cloudfoundry.org/devguide/services/service-keys.html).
+```
+
+Run the following command to set up the environment variables in your system:
+
+```bash
+export $(xargs < .env)
+```
+### Usage:
```

Comment:
I suggest to put the explanation about the 2 modes in the introductory parts and keep the step instructions leaner. To do this, delete lines 50-60.

```suggestion
``` #

Reply:
updated